frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts

```diff
@@ -85,10 +94,54 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
         );
         this.firstVehiclesCount -= firstVehiclesCountReducedBy;
 
+        const vehicleTemplates = selectStateSnapshot(
+            selectVehicleTemplates,
+            this.store
+        );
+        const vehicleTemplatesById = Object.fromEntries(
+            vehicleTemplates.map((template) => [template.id, template])
+        );
+
+        const materialTemplates = selectStateSnapshot(
+            selectMaterialTemplates,
+            this.store
+        );
+        const personnelTemplates = selectStateSnapshot(
+            selectPersonnelTemplates,
+            this.store
+        );
+
+        const sortedAlarmGroupVehicles = StrictObject.values(
+            alarmGroup.alarmGroupVehicles
+        ).sort((a, b) => a.time - b.time);
```

Comment:
Not really related to the change, but isn't this kind of weird? If we have a `firstVehiclesCount` of `2` but three vehicles arriving at the same time, wouldn't it be basically uncontrollable which vehicles arrive at which transfer point?

Reply:
`Object.keys` returns the keys in a deterministic order and `JSON.stringify`/`JSON.parse` preserves this order. `sort` is stable, so if multiple vehicles have the same arrival time, the cutoff will be consistently defined by the order in which the vehicles have been added to the alarm group.

Reply:
I could change the new implementation to explicitly sort by, for example, UUID. But we would still carry this behavior from the migration, so I decided to do as few changes as possible.

Reply:
Yeah, I don't think this is something we should change in this PR. And yes, it is deterministic, but from an end-users point of view it is not plausible why it happens the way it does.

Reply:
I've made #1068 for this.